docs/differences-from-v1.md

```diff
@@ -4,41 +4,80 @@ Upgrading from FOSSA CLI 1.x to 2.x is a major breaking change. For most users,
 
-### "Modules" are now "analysis targets"
-
-In 1.x, the CLI was configured by "modules", which were entrypoints to user programs that the CLI should analyze.
+### New Build Manager Supports
 
-In 2.x, we've renamed "modules" to "analysis targets", and simplified how analysis targets are defined.
+FOSSA 2.x supports following new build managers and languages:
 
-We renamed analysis targets to avoid confusing naming conflicts with existing language-specific "modules". Analysis targets are no longer defined by multiple, confusingly similar fields (e.g. "Dir" vs. "Path"), and are now simple `(type, target)` tuples.
+- [Swift Package Manager (for swift)](strategies/ios/swift.md)
+- [Pub (for flutter and dart)](strategies/dart.md)
+- [Poetry (for Python)](strategies/python.md)
+- [Mix (for Elixir)](strategies/elixir.md)
+- [Fortran Package Manager (for Fortran)](strategies/fortran.md)
 
 ### Automatic analysis target discovery
 
-FOSSA CLI 2.x now does automatic analysis target discovery when you run `fossa analyze`.
+FOSSA CLI 2.x now does automatic analysis target discovery when you run `fossa analyze`. This means that `fossa init` is no longer required. `fossa init` is now a no-op that emits a warning, and will be removed in a future release.
 
-This means that `fossa init` is no longer required. `fossa init` is now a no-op that emits a warning, and will be removed in a future release.
+### New fossa-deps configuration support
 
-### Automatic analysis target strategy selection
+With .fossa-deps.yml file, 2.x supports:
+
+- License scanning vendor dependencies 
+- Manually specifying dependency for analysis by its source code archive web address (e.g. https://my-deps-source/v1.zip)
+- Manually specifying dependency for analysis by it's name and license (e.g. my-custom-dep with MIT licence)
+- Manually specifying dependency for analysis by its name and dependency type (e.g. pip dependency: request)
+
+This is very useful when working with a package manager that is not supported, or when you have a custom and nonstandard dependency management solution. The FOSSA CLI will automatically read a fossa-deps.yml or a fossa-deps.json file in the root directory (usually the current working directory) when fossa analyze is run and parse dependencies from it. These dependencies will be added to the dependencies that are normally found when fossa analysis is run in the directory.
+
+Please refer to [fossa-deps](userguide.md#manually-specifying-dependencies) documentation for more details.
 
-In 1.x, modules could be manually configured with "strategies" that specified _how_ a module should be analyzed. While `fossa init` attempted to choose the best strategy, manual intervention was sometimes required depending on the project's setup.
+### Automatic analysis target strategy selection
```

Comment:
I think this is basically a duplicate section to line 61. If it isn't you should combine them.